README.md

```diff
@@ -7,11 +7,13 @@ Before working on projects in this repo, we suggest taking time to follow [the p
 ## Solutions
 
 At the moment, these are the solutions in this repo:
-
-1) **[cloudformationTemplates/asyncCustomerChatUX](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/cloudformationTemplates/asyncCustomerChatUX)**
+ 1) **Get started with Chat** - **[CloudFormationTemplates/startChatContactAPI](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/cloudformationTemplates/startChatContactAPI)**
+    Use this solution to create a web widget with Amazon Connect Chat.
```

Comment:
This solution does not create a widget. This one is just the API and has the customer manually adjust it.

Also, I intentionally put the async solution first as it requires no effort past the stack deployment and so I want customers to use that before the API one

Reply:
Yup. I intentionally put the startChat one at the top is because, for customers who aren't as familiar with AWS understanding that architecture is another thing they have to learn. 
For customers who want to get started with a solution in their web app, with an index.html the startChat solution is much simpler.